comment_iowriter.go

```diff
@@ -142,3 +144,135 @@ func (cw *RDJSONCommentWriter) Flush(_ context.Context) error {
 	}
 	return nil
 }
+
+var _ CommentService = &SARIFCommentWriter{}
+
+// SARIFCommentWriter
+type SARIFCommentWriter struct {
+	w        io.Writer
+	comments []*Comment
+	toolName string
+}
+
+func NewSARIFCommentWriter(w io.Writer, toolName string) *SARIFCommentWriter {
+	return &SARIFCommentWriter{w: w, toolName: toolName}
+}
+
+func (cw *SARIFCommentWriter) Post(_ context.Context, c *Comment) error {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method SARIFCommentWriter.Post should have comment or be unexported
<!-- __reviewdog__:ChBhODc5N2IxODIwMzU5N2QxEg9nb2xpbnQtY2lyY2xlY2k= -->